modules/dnn/src/tensorflow/tf_graph_simplifier.cpp

```diff
@@ -829,12 +829,19 @@ void RemoveIdentityOps(tensorflow::GraphDef& net)
             IdentityOpsMap::iterator it = identity_ops.find(input_op_name);
 
             if (it != identity_ops.end()) {
+                std::vector<IdentityOpsMap::iterator> itVec;
```

Comment:
What is observed size of this container for valid models?
If it is a large (>50 items) then we should try `std::set` here.

---

itVec -> loopCheck / duplicateCheck / other descriptive name.

Reply:
Thanks for your code review. 
Different models may have different `itVec` sizes. Since it just put the item in `identity_ops` to `itVec`, the size of `identity_ops` is the max number.
From my point of view, the maximum size of `identity_ops` most should be less than 50.